src/modules/tccpp/components/the-button.ts

```diff
@@ -72,6 +75,61 @@ export default class TheButton extends BotComponent {
         button_scoreboard: the_button_scoreboard_entry;
     }>();
 
+    override async setup(commands: CommandSetBuilder) {
+        commands.add(
+            new TextBasedCommandBuilder("wololosetupthebutton", EarlyReplyMode.none)
```

Comment:
It would be good to make these non-slash so they're harder to accidentally run from a bad tab completion

Reply:
Yeah that was my intention. My understanding is that they are not slash commands unless `.set_slash(true)`? I can add explicit `.set_slash(false)`, but I don't think Ive seen any other code do that, so I assumed that we generally just leave it out.